NEWS

```diff
@@ -1,3 +1,20 @@
+17-May-2018 IGNITION v0.25.0
+
+  Features:
+
+     - Blackbox test can now be run in parallel
+
+  Changes:
+
+     - Remove Oracle Cloud Infrastructure support
+
+  Bug Fixes:
+
+     - Fix issue where Ignition would leak a temporary file when appending to
+       an existing file.
+     - Fix multiple blackbox test validation errors
+     - Fix v1 config validation to match the v2.0.0+ config validation
```

Comment:
This is pretty unclear.  `Fix v1 config parsing to return ErrUnknownVersion if version is unrecognized`?